blaze/expr/collections.py

```diff
@@ -424,10 +424,35 @@ def join(lhs, rhs, on_left=None, on_right=None, how='inner'):
 
 join.__doc__ = Join.__doc__
 
+class IsIn(Expr):
```

Comment:
The expression I had in mind works like this:

```python
In [1]: def isin(a, b):
   ...:     return [el in b for el in a]
   ...:

In [2]: isin([1, 2, 3], [1, 2, 4, 5, 6])
Out[2]: [True, True, False]
```
That's the python sequence version. The pandas version for `Series` objects would look like this:

```python
In [3]: s = pd.Series([1, 2, 3])

In [4]: s.isin([1, 2, 4, 5, 6])
Out[4]:
0     True
1     True
2    False
dtype: bool
```
So you might have a recipe like this in `python.py`:

```python
@dispatch(IsIn, Sequence)
def compute_up(expr, data, **kwargs):
    key = expr._key
    return [el in key for el in data]
```